.changeset/in-tx-guard-row-lock.md

```
@@ -0,0 +1,16 @@
+---
+"@linchkit/core": minor
+---
+
+Close the residual record-state guard TOCTOU window with a row-level lock (#470).
+
+#469 moved `block` / `require_approval` guard-rule evaluation inside the write
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)